src/js/gutenberg-plugins/content-resizing-plugin.js

```diff
@@ -330,6 +360,26 @@ const ContentResizingPlugin = () => {
 					</tbody>
 				</table>
 			</div>
+			<p>
+				{ __(
+					'None of these suggestions are ideal, please',
```

Comment:
I agree with your thought here: https://github.com/10up/classifai/issues/748#issuecomment-2137301539. To me this reads as if something went wrong and we're giving an error message. It also doesn't make sense if someone has it set to only show a single result (`suggestions` vs `suggestion`).

I also wonder if a button makes more sense here, as that would stand out more, especially when the disable functionality is turned on, we already show a link here:

![Content resizing modal](https://github.com/10up/classifai/assets/916738/aeebb575-65fd-48cd-88bb-93cb704308fb)

Something like this:

![Refresh results in modal](https://github.com/10up/classifai/assets/916738/fba76bfd-3060-4d20-924a-c58398758999)

I also think `Retry expanding/condensing the text` looks good here, so I'm fine with either text:

![Retry button in modal](https://github.com/10up/classifai/assets/916738/5bface83-3cdc-469a-a2ae-84ed3f55af59)